src/Symfony/Component/Mailer/Bridge/Mailchimp/Http/MandrillTransport.php

```diff
@@ -41,7 +43,7 @@ protected function doSend(SentMessage $message): void
         $response = $this->client->request('POST', self::ENDPOINT, [
             'json' => [
                 'key' => $this->key,
-                'to' => $this->stringifyAddresses($envelope->getRecipients()),
+                'to' => $this->getRecipients($envelope),
```

Comment:
Reading the docs, it looks like that the `To` parameter is optional. I think it's better to remove it and let Mandrill do it itself.

Reply:
Yes, but if we do not include it, then I think `Symfony\Component\Mailer\EventListener\EnvelopeListener` will not work.